src/parser/condition.rs

```diff
@@ -173,7 +183,7 @@ fn build_condition_recursively(name: &str, obj: &Value) -> Result<ConditionValue
 
                 ConditionValue::Or(v)
             }
-            "!Condition" | "Condition" => {
+            Some("Condition") => {
```

Comment:
The sample here says that `!Condition` is real:

```yaml
  CreateBucketPolicy: !And 
    - !Condition IsProduction
    - !Condition CreateBucket
```

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/conditions-section-structure.html#conditions-section-structure-examples